scheds/rust/scx_p2dq/src/bpf/main.bpf.c

```diff
@@ -512,47 +513,83 @@ static s32 pick_idle_cpu(struct task_struct *p, struct task_ctx *taskc,
 	if (wake_flags & SCX_WAKE_SYNC) {
 		struct task_struct *current = (void *)bpf_get_current_task_btf();
 		struct task_ctx *cur_taskc = lookup_task_ctx_may_fail(current);
```

Comment:
Is it worth maybe using some different names here? I found the new code quite hard to follow until I realised that "current" here means "waker".

Reply:
Yeah, the naming is not great. I'm probably going to do another round of refactoring in this path soon.